components/webui/imports/api/search/constants.js

```diff
@@ -28,6 +28,10 @@ const isSearchSignalQuerying = (s) => (
         SEARCH_SIGNAL.RESP_QUERYING,
     ].includes(s)
 );
+const isDisablingUserInput = (s) => (
```

Comment:
How about `isOperationInProgress`?

Reply:
Much better.

Reply:
Maybe `isSearchInProgress`?

Reply:
Technically isSearchSignalReq includes REQ_CLEARING which is not really search, right?

Reply:
Right, right. I was hoping for consistency with the other checker functions.
Let's use `isOperationInProgress` for now.

Reply:
Gotcha, gotcha. I guess we can rename things more nicely in future PRs.